code/src/Form/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './form.css'
+import { API_URL } from '../utils/urls'
+
+export const Form = ({ message, setMessage, thougths, setThougths }) => {
+
+    const handleFormSubmit = (e) => {
+        e.preventDefault()
+
+        const options = {
+            method: 'POST',
+            body: JSON.stringify({ message: message }),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        }
+
+        fetch(API_URL, options)
+            .then((res) => res.json())
+            .catch(error => console.error('Error:', error))
+            .then((newThought) => {
+                setThougths([newThought, ...thougths])
+                setMessage("")
+            })
+    }
+
+    let charactersRemaining = 140 - message.length;
+
+    return (
+        <form className="form" onSubmit={handleFormSubmit}>
+            <input
```

Comment:
Do you think we could use <textarea> instead of input in order to be able to see a long message?